scripts/build.ts

```diff
@@ -19,7 +19,12 @@ fileReplace('dist/es/Constants.js', '__STANZAJS_VERSION__', Pkg.version);
 
 Child('npm run compile:rollup');
 
-Child('mkdir dist/npm');
+if (!FS.existsSync("dist")) {
```

Comment:
This first `dist` mkdir is unnecessary, as the original call was `mkdir` with no `-p`